packages/core/src/utils.ts

```diff
@@ -14,6 +15,43 @@ export const getKeys = (contractName: string) => {
   }
 };
 
+export const getFile = (fileWithExtension: string) => {
+  try {
+    return (
+      fs.readFileSync(`${ZENCODE_DIR}/${fileWithExtension}`).toString() || null
+    );
+  } catch (e) {
+    return null;
+  }
+};
+
+/**
+ *  Returns zencode from a contract name
+ *  @param contractName 
+ *  @returns {Zencode}
+ */
+export const getContractByContractName = (contractName: string) : Zencode => {
+  return Zencode.byName(contractName, ZENCODE_DIR);
+};
+
+/**
+ *  Returns zencode from a partial path
+ *  @param path 
+ *  @returns {Zencode}
+ */
+export const getContractFromPath = (path: string) : Zencode => {
+  return Zencode.fromPath(`${ZENCODE_DIR}/${path}`);
+};
+
+/**
+ *  Returns string representing a .yml file
+ *  @param ymlName 
+ *  @returns {string}
+ */
+export const getYml = (ymlName: string) => {
+  return fs.readFileSync(`${ZENCODE_DIR}/${ymlName}.yml`).toString() || null;
```

Comment:
can we have .yml not harcoded... many times I saw also people using .yaml as a file extension... would like to prevent errors just by missing an `a` for people, back to us with "restroom is not working" 🤣